server/server.go

```diff
@@ -159,7 +159,8 @@ func (server *Server) getDatabaseConfig() (string, string, bool, bool) {
 	}
 	databaseDropOnCreate := config.GetBool("database/drop_on_create", false)
 	databaseCascade := config.GetBool("database/cascade_delete", false)
-	return databaseType, databaseConnection, databaseDropOnCreate, databaseCascade
+	databaseAutoMigrate := config.GetBool("database/auto_migrate", true)
```

Comment:
I think it would be also nice to have auto_create flag, which would disable table creation and initial data insertion. This way, we can be sure, we don't mixup external migration with automatically created db.